src/test/resources/log4j.properties

```diff
@@ -0,0 +1,5 @@
+log4j.rootLogger=DEBUG, stdout
```

Comment:
Shouldn't `stdout` have the corresponding `log4j.appender.stdout.target=System.out`?

Reply:
I guess the name is just a little confusion... we want to capture debug-level logs, not what's written to System.out and print them to the console/stdout...

Or do I miss understand your comment?

Reply:
I'm just referring to the name as is that has no corresponding `appender` defined whether it's `System.out` or not. I thought it's required to have a `log4j.appender.[name].target` entry defined.

Reply:
Default target is `System.out` -- no need to specify.